src/test/java/Typos.java

```diff
@@ -0,0 +1,38 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class Typos {
+
+        WebDriver driver;
+        private static final String HEROCU_APP_URL = "http://the-internet.herokuapp.com/typos";
+
+        @BeforeMethod
+        public void initMethod() {
+            System.setProperty("webdriver.chrome.driver", "src/test/resources/chromedriver.exe");
+            driver = new ChromeDriver();
+            driver.manage().window().maximize();
+        }
+        @Test
+        public void testTypos() {
+            // Open site http://the-internet.herokuapp.com/typos
+            driver.get(HEROCU_APP_URL);
+            // Find text by xpath
+            WebElement typos = driver.findElement(By.xpath("//*[@id=\"content\"]/div/p[2]"));
```

Comment:
be more specific with variables names. better to use name like: WebElement textField or smth like that